pages/en/lb3/datasources.json.md

```diff
@@ -168,12 +171,11 @@ The additional files can override the top-level data-source options with string
 
 Example data sources:
 
+In the JSON, the key is the datasource name and the value is the configuration object to pass to `app.dataSource(name, config)`.
+
 {% include code-caption.html content="datasources.json" %}
 ```javascript
 {
-  // the key is the datasource name
-  // the value is the config object to pass to
-  // app.dataSource(name, config).
   db: {
     connector: 'memory'
   }
```

Comment:
content of this file is greatly duplicated with content in Environment-specific-configuration.html#data-source-configuration, which will make the whole difficult to maintain. Hence, cross-linking would be preferable.